webanno-diag/src/main/java/de/tudarmstadt/ukp/clarin/webanno/diag/repairs/ReattachFeatureAttachedSpanAnnotationsAndDeleteExtrasRepair.java

```diff
@@ -102,7 +102,7 @@ else if (candidate != anno) {
             
             // Delete those the extra candidates that are not properly attached
             if (!toDelete.isEmpty()) {
-                toDelete.forEach(fs -> aCas.removeFsFromIndexes(fs));
```

Comment:
In all places like this I am not sure anymore if this can cause this JVM bug you mentioned before. If you think its fine merge it. Otherwise I can revert it.